lib/components/src/Loader/Loader.tsx

```diff
@@ -0,0 +1,26 @@
+import { keyframes, styled } from '@storybook/theming';
+
+const spin = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+export const Loader = styled.div`
```

Comment:
Could you please change this to the object style of creating components.

We do not use the emotion babel transform, because we re-export emotion's exports, the babel-transform can't detect that.

Besides the fact this string will appear literally in the bundle, the object style has preference for a multitude of reason, one of which is it's how most of the storybook code is written.

```suggestion
export const Loader = styled.div(({ theme }) => ({
  animation: `${spin} 1s linear infinite`,
  borderRadius: '50%',
  height: 48,
  left: 'calc(50% - 24px)',
  position: 'absolute',
  top: 'calc(50% - 24px)',
  width: 48,
  zIndex: 4,
  border: `3px solid ${theme.color.secondary}`,
  borderTop: `3px solid transparent`,
});
```

Reply:
I tried to make the code-suggestion to replace then multiple lines, but couldn't get that. it should obviously replace the whole thing, not just the first line 😄

Reply:
Done 👍